### ISSUE TITLE
vagrant instance fails to load

### ISSUE THREAD

**Issue by rajbot** [internetarchive/openlibrary#211]
When checking out the repo and typing `vagrant up`, the bootstrap process will fail with:

```
fatal: Not a git repository: /vagrant/.git/modules/vendor/acs4_py
```

**Comment by anandology** [comment]
@rajbot that issue is fixed in 02c500c94594f1096d38cb43fa506f83c13ce517. 
Please try now and let me know if you still see any issues.

**Comment by tfmorris** [comment]
`vagrant up` works for me on OS X.  I haven't recently tested it on Windows.
